src/App.js

```diff
@@ -58,8 +60,8 @@ export default function App() {
     init();
   }, []);
 
-  const getTemp = async () => {
-    const { data } = await getTemperature(inputText);
+  const getTemp = async (city) => {
```

Comment:
looks like the parameter is accepting coordinates not city

Reply:
resolved